frontend/packages/dev-console/src/components/__tests__/EmptyState.spec.tsx

```diff
@@ -1,23 +0,0 @@
-import * as React from 'react';
-import configureMockStore from 'redux-mock-store';
-import { Map as ImmutableMap } from 'immutable';
-import { shallow } from 'enzyme';
-import ConnectedEmptyStateComponent from '../EmptyState';
-import { getStoreTypedComponent } from '../../test/test-utils';
-
-describe('EmptyState', () => {
```

Comment:
Doesn't really test anything, and fails when upgrading to newer `react-redux` version.